api/rpc/perms/permissions.go

```diff
@@ -28,9 +32,21 @@ func (j *JWTPayload) MarshalBinary() (data []byte, err error) {
 
 // NewTokenWithPerms generates and signs a new JWT token with the given secret
 // and given permissions.
-func NewTokenWithPerms(signer jwt.Signer, perms []auth.Permission) ([]byte, error) {
+func NewTokenWithPerms(signer jwt.Signer, perms []auth.Permission, ttl time.Duration) ([]byte, error) {
```

Comment:
ok sorry missed this one but can you make a separate public method that allows for setting ttl and make this `NewTokenWithPerms` rely on it and not set a TTL ? 

so `NewTokenWithPerms` and `NewTokenWithTTL`

Reply:
Ah, okay. Now it clear. Have added a separate function with TTL.